fns-hl7-pipeline/README.md

```diff
@@ -1 +1,93 @@
-HL7 pipeline functions project: Receiver-Debatch, Validate, Transform, Validate Content.
\ No newline at end of file
+# CDC Data Exchange HL7
+
+HL7 pipeline functions project: json-lake, segments-transformer, mmg-based-transformer, mmg-sql-transformer, mmg-validator, receiver-debatcher, redactor, structure-validator. 
```

Comment:
I think it would be better to list the pipeline functions in the order in which they are executed:
receiver-debatcher, redactor, structure-validator, mmg-validator, mmg-based-transformer, mmg-sql-transformer, lake-segments-transformer, json-lake-transformer.

Reply:
good point,  I will change that.